Rules/CTF/Scripts/CTF_Trading.as

```diff
@@ -100,7 +101,7 @@ void onPlayerDie(CRules@ this, CPlayer@ victim, CPlayer@ killer, u8 customData)
 		{
 			if (killer !is victim && killer.getTeamNum() != victim.getTeamNum())
 			{
-				killer.server_setCoins(killer.getCoins() + coinsOnKillAdd);
+				killer.server_setCoins(killer.getCoins() + coinsOnKillAdd * (1+ (killer.get_u8("killstreak") - 1)*killstreakFactor));
```

Comment:
This can be simplified.
```suggestion
				killer.server_setCoins(killer.getCoins() + coinsOnKillAdd * killer.get_u8("killstreak") * killstreakFactor);
```